examples/chip-spv/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+#=============================================================================
+#   CMake build system files
+#
+#   Copyright (c) 2023 Michal Babej / Intel Finland Oy
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a copy
+#   of this software and associated documentation files (the "Software"), to deal
+#   in the Software without restriction, including without limitation the rights
+#   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#   copies of the Software, and to permit persons to whom the Software is
+#   furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included in
+#   all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#   THE SOFTWARE.
+#
+#=============================================================================
+
+set(TS_NAME "chip-spv")
+set(TS_BASEDIR "${TESTSUITE_BASEDIR}/${TS_NAME}")
+set(TS_BUILDDIR "${TS_BASEDIR}/src/${TS_NAME}-build")
+set(TS_SRCDIR "${TS_BASEDIR}/src/${TS_NAME}")
+
+if(NOT HAVE_GIT)
+  message(STATUS "Disabling testsuite ${TS_NAME}, requires git to checkout sources")
+  return()
+endif()
+
+if(ENABLE_ICD AND (NOT OPENCL_LIBRARIES))
+  message(STATUS "Disabling ${TS_NAME}, can't find OpenCL library")
+  return()
+endif()
+
+if(NOT ENABLE_LLVM)
+  message(STATUS "Disabling ${TS_NAME}, needs ENABLE_LLVM=ON")
+  return()
+endif()
+
+if(ENABLE_HOST_CPU_DEVICES AND (ENABLE_CONFORMANCE))
+  message(WARNING "CHIP-SPV + CPU driver with ENABLE_CONFORMANCE=ON will fail some tests")
```

Comment:
Why is this? I'm thinking we should ENABLE_CONFORMANCE=ON by default as soon as we get the (long pending) conformance submission done.

Reply:
Because enabling conformance disables some features (like image formats) in the CPU driver. We have some buggy/incomplete features, and if we enable them, the CTS tries to test it and it fails. OTOH, CHIP-SPV has tests that need those features.